google_books/src/components/BookCard/BookCard.jsx

```diff
@@ -0,0 +1,36 @@
+import styles from "./BookCard.module.scss";
+
+const BookCard = ({
+  id,
+  image,
+  author,
+  title,
+  description,
+  setBookID,
+  setShowModal,
+}) => {
```

Comment:
`({ book })` 

The reason it's okay to pass an entire book object here, is because something called `BookCard` already has an implicit dependency on a `book` record